server/backend/quiz/urls.py

```
@@ -9,4 +9,5 @@
     path('submit-quiz/', SubmitQuizAPIView.as_view(), name='submit-quiz'),
     path('add/', AddQuizView.as_view(), name='add_quiz'),
     path('add-question/', AddQuestionView.as_view(), name='add-question'),
+    path('update-delete/<int:category_id>/', UpdateDeleteQuizView.as_view(), name='update-delete-quiz')
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)